CHANGES.md

```diff
@@ -32,6 +37,8 @@ To be released.
 
 ### Behavioral changes
 
+ -  `Block<T>.Transactions` will be ordered using a different scheme once
+    the protocol version is bumped to `3`.  [[#1322], [#1323], [#1518]]
```

Comment:
As far as I know, `ActionEvaluator<T>.OrderTxsForEvaluation()` does not affect `Block<T>.Transactions` proprerty as data.  (Actually, `Block<T>.Transactions` are ordered by `TxId`.)  It only affect the evaluation order when states are determined.

Reply:
As per suggestion, rephrased the changelog a bit. 😗